pkg/k2-asr/src/huggingface.py

```diff
@@ -41,17 +23,46 @@ def load_model(device="cpu", precision="fp32"):
     Returns:
       sherpa_onnx.OfflineRecognizer
     """
-    if precision not in HF_REPO_FILES:
+
+    if language == "ja":
+        hf_repo_id = "reazon-research/reazonspeech-k2-v2"
+        epochs = 99
+    elif language == "multi":
+        hf_repo_id = "reazon-research/k2-multi-ja-en"
+        epochs = 35
+
```

Comment:
Add `else` clause here and raise a `ValueError` exception when
an unknown language name is passed.

See the theck for `precision` below for more details.